wooey/templates/wooey/scripts/script_view.html

```diff
@@ -18,6 +18,16 @@ <h3 id="wooey-script-title">{{ script.script_name }}</h3>
 
   <div class="container-fluid">
 
+        {% if script.documentation %}
+        <div class="panel panel-default">
+            <div class="panel-heading">Documentation  <a class="icon icon-collapse" data-toggle="collapse" data-target="#collapse-metadata" href="#collapse-metadata"></a></div>
```

Comment:
Can you wrap this in a trans tag? Also, is the anchor supposed to be empty or is Documentation supposed to be in it?